packages/hdwallet-tallyho/src/tallyho.ts

```diff
@@ -92,6 +92,7 @@ export class TallyHoHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsOptimism = false;
   readonly _supportsPolygon = true;
   readonly _supportsGnosis = false;
+  readonly _supportsArbitrum = true;
```

Comment:
FWIW, doesn't seem live yet https://taho.upvoty.com/b/taho-roadmap/arbitrum-support/

Reply:
It's marked live and showing in the wallet:
![image](https://github.com/shapeshift/hdwallet/assets/35275952/66229f2a-0f0d-4e37-9f4a-9ac0c861e796)
![image](https://github.com/shapeshift/hdwallet/assets/35275952/822eb787-ddab-427f-a9f4-d40e10ceb81e)